lib/ionic_notification.rb

```diff
@@ -27,6 +31,7 @@ def self.setup
   end
 end
 
+require 'ionic_notification/logger'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.